tests/floating-point/fft/harness.fil

```diff
@@ -103,7 +151,23 @@ comp main<'G: 1>(
   spiral_bundle{13} = spdft.Y13;
   spiral_bundle{14} = spdft.Y14;
   spiral_bundle{15} = spdft.Y15;
+  spiral_bundle{16} = spdft.Y16;
+  spiral_bundle{17} = spdft.Y17;
+  spiral_bundle{18} = spdft.Y18;
+  spiral_bundle{19} = spdft.Y19;
+  spiral_bundle{20} = spdft.Y20;
+  spiral_bundle{21} = spdft.Y21;
+  spiral_bundle{22} = spdft.Y22;
+  spiral_bundle{23} = spdft.Y23;
+  spiral_bundle{24} = spdft.Y24;
+  spiral_bundle{25} = spdft.Y25;
+  spiral_bundle{26} = spdft.Y26;
+  spiral_bundle{27} = spdft.Y27;
+  spiral_bundle{28} = spdft.Y28;
+  spiral_bundle{29} = spdft.Y29;
+  spiral_bundle{30} = spdft.Y30;
+  spiral_bundle{31} = spdft.Y31;
```

Comment:
Another reason to support the whole "predictable bundle lowering" for external interfaces is so that we don't have to write these unrolled assignments by hand (I do this for Aetherling too)